cinema/migrations/0001_initial.py

```diff
@@ -1,4 +1,4 @@
-# Generated by Django 4.0.2 on 2022-05-03 13:41
```

Comment:
No. You shouldn't modify existing migration. You should create a new one instead. It is a good practice to include one migration per app per PR in Django projects since it helps to avoid migrations folder overload in the future. Represent all changes done to models in one migration file per app per PR, please

Reply:
Do not modify this file. Please revert changes made here. 
Instead, create a new one migration file and include it to PR